Index: developers-reference.sgml
===================================================================
RCS file: /org/cvs.debian.org//cvs/debian-doc/ddp/manuals.sgml/developers-reference/developers-reference.sgml,v
retrieving revision 1.257
diff -u -r1.257 developers-reference.sgml
--- developers-reference.sgml	26 Jan 2005 09:21:37 -0000	1.257
+++ developers-reference.sgml	1 Feb 2005 19:44:23 -0000
@@ -52,6 +52,11 @@
 the &debian-formal; distribution or on the World Wide Web at <url
 id="&url-gpl;" name="the GNU web site">.  You can also obtain it by
 writing to the &fsf-addr;.
+<!-- BEGIN-PDFLINK -->
+       <p>
+If you want to print this reference, you should use the <url
+id="&url-pdf-reference;" name="pdf version">.
+<!-- END-PDFLINK -->
 
     <toc detail="sect1">
 
