--- maint-guide.zh-cn.dbk-orig	2011-03-20 18:53:39.520948369 +0900
+++ maint-guide.zh-cn.dbk	2011-03-20 18:38:34.048950323 +0900
@@ -19,6 +19,9 @@
 
 
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>Aron</firstname> <surname>Xu</surname><contrib>(简体中文翻译)</contrib></personname><email>happyaron.xu@gmail.com</email></othercredit>
+<othercredit role="translator"><personname><surname>李</surname><firstname>凌</firstname> <contrib>(简体中文翻译)</contrib></personname><email>lilingv@gmail.com</email></othercredit>
+<othercredit role="translator"><personname><surname>郑</surname><firstname>原真</firstname> <contrib>(简体中文翻译)</contrib></personname><email>ycheng@slat.org</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 </authorgroup>
 <releaseinfo>version &docversion;</releaseinfo>
