--- maint-guide.es.dbk-orig	2011-04-01 02:04:19.339309863 +0900
+++ maint-guide.es.dbk	2011-04-01 02:03:33.967387343 +0900
@@ -18,11 +18,12 @@
 
 
 
-
-
-
-
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>Javier</firstname> <othername>Fernández-Sanguino</othername> <surname>Peña</surname><contrib>(Traducción al español)</contrib></personname><email>jfs@debian.org</email></othercredit>
+<othercredit role="translator"><personname><firstname>David</firstname> <surname>Martínez</surname><contrib>(Traducción al español)</contrib></personname><email>ender@debian.org</email></othercredit>
+<othercredit role="translator"><personname><firstname>Ana</firstname> <othername>Beatriz Guerrero</othername> <surname>López</surname><contrib>(Traducción al español)</contrib></personname><email>ana@debian.org</email></othercredit>
+<othercredit role="translator"><personname><firstname>Francisco</firstname> <othername>Javier</othername> <surname>Cuadrado</surname><contrib>(Traducción al español)</contrib></personname><email>fcocuadrado@gmail.com</email></othercredit>
+<othercredit role="translator"><personname><firstname>Innocent</firstname> <othername>De</othername> <surname>Marchi</surname><contrib>(Traducción al español)</contrib></personname><email>tangram.peces@gmail.com</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 <!-- dummy -->
 <!-- dummy -->
