--- maint-guide.de.dbk-orig	2011-04-01 01:30:35.015309955 +0900
+++ maint-guide.de.dbk	2011-04-01 01:30:35.027311481 +0900
@@ -19,6 +19,9 @@
 
 
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+   <othercredit role="translation"><personname><firstname>Tobias</firstname> <surname>Quathamer</surname><contrib>(Deutsch-Übersetzung)</contrib></personname><email>toddy@debian.org</email></othercredit>
+   <othercredit role="translation"><personname><firstname>Erik Schanze</firstname> <surname>Schanze</surname><contrib>(Deutsch-Übersetzung)</contrib></personname><email>mail@erikschanze.de</email></othercredit>
+   <othercredit role="translation"><personname><firstname>Eduard</firstname> <surname>Bloch</surname><contrib>(Deutsch-Übersetzung)</contrib></personname><email>blade@debian.org</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 </authorgroup>
 <releaseinfo>Version &docversion;</releaseinfo>
