--- maint-guide.it.dbk-orig	2011-04-01 02:04:26.751309999 +0900
+++ maint-guide.it.dbk	2011-04-01 02:03:45.708405693 +0900
@@ -18,11 +18,10 @@
 
 
 
-
-
-
-
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>Calogero</firstname> <othername>Lo</othername> <surname>Leggio</surname><contrib>(Traduzione italiana)</contrib></personname><email>kalos@nerdrug.org</email></othercredit>
+<othercredit role="translator"><personname><firstname>Jacopo</firstname> <surname>Reggiani</surname><contrib>(Traduzione italiana)</contrib></personname><email>jacopo.reggiani@gmail.com</email></othercredit>
+<othercredit role="translator"><personname><firstname>Francesco</firstname> <othername>P.</othername> <surname>Lovergine</surname><contrib>(Traduzione italiana)</contrib></personname></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 <!-- dummy -->
 <!-- dummy -->
