--- maint-guide.ca.dbk-orig	2011-03-31 01:47:36.424081337 +0900
+++ maint-guide.ca.dbk	2011-03-31 01:49:03.808157278 +0900
@@ -19,6 +19,7 @@
 
 
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>Innocent</firstname> <othername>De</othername> <surname>Marchi</surname><contrib>(Traducción al español)</contrib></personname><email>tangram.peces@gmail.com</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 </authorgroup>
 <releaseinfo>version &docversion;</releaseinfo>
