--- maint-guide.ca.dbk-orig	2011-04-01 02:04:11.924310046 +0900
+++ maint-guide.ca.dbk	2011-04-01 02:02:45.856383298 +0900
@@ -18,11 +18,8 @@
 
 
 
-
-
-
-
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>Innocent</firstname> <othername>De</othername> <surname>Marchi</surname><contrib>(Traducció al
+català)</contrib></personname><email>tangram.peces@gmail.com</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 <!-- dummy -->
 <!-- dummy -->
