--- maint-guide.fr.dbk-orig	2011-04-01 01:45:32.607309921 +0900
+++ maint-guide.fr.dbk	2011-04-01 01:45:32.619312216 +0900
@@ -21,6 +21,10 @@
 
 
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+   <othercredit role="translation"><personname><firstname>Frédéric</firstname> <surname>Dumont</surname><contrib>(traduction française)</contrib></personname><email>frederic.dumont@easynet.be</email></othercredit>
+   <othercredit role="translation"><personname><firstname>Mohammed</firstname> <othername>Adnène</othername><surname>Trojette</surname><contrib>(traduction française)</contrib></personname><email>adn+deb@diwi.org</email></othercredit>
+   <othercredit role="translation"><personname><firstname>David</firstname> <surname>Prévot</surname><contrib>traduction française</contrib></personname><email></email></othercredit>
+   <othercredit role="translation"><personname>les membres de la liste debian-l10n-french@lists.debian.org</personname><email>debian-l10n-french@lists.debian.org</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 </authorgroup>
 <releaseinfo>version &docversion;</releaseinfo>
