--- maint-guide.it.dbk-orig	2011-04-01 01:45:36.307309991 +0900
+++ maint-guide.it.dbk	2011-04-01 01:45:36.323327388 +0900
@@ -21,6 +21,9 @@
 
 
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>Calogero</firstname> <othername>Lo</othername> <surname>Leggio</surname><contrib>(Traduzione italiana)</contrib></personname><email>kalos@nerdrug.org</email></othercredit>
+<othercredit role="translator"><personname><firstname>Jacopo</firstname> <surname>Reggiani</surname><contrib>(Traduzione italiana)</contrib></personname><email>jacopo.reggiani@gmail.com</email></othercredit>
+<othercredit role="translator"><personname><firstname>Francesco</firstname> <othername>P.</othername> <surname>Lovergine</surname><contrib>(Traduzione italiana)</contrib></personname></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 </authorgroup>
 <releaseinfo>version &docversion;</releaseinfo>
