--- maint-guide.ja.dbk-orig	2011-04-01 02:04:30.464309964 +0900
+++ maint-guide.ja.dbk	2011-04-01 02:03:51.495389236 +0900
@@ -18,11 +18,12 @@
 
 
 
-
-
-
-
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>望</firstname> <surname>倉澤</surname><contrib>(日本語訳)</contrib></personname><email>nabetaro@debian.or.jp</email></othercredit>
+<othercredit role="translator"><personname><firstname>雄介</firstname> <surname>八津尾</surname><contrib>(日本語訳)</contrib></personname><email>yyatsuo@gmail.com</email></othercredit>
+<othercredit role="translator"><personname><firstname>洋平</firstname> <surname>佐々木</surname><contrib>(日本語訳)</contrib></personname><email>uwabami@gfd-dennou.org</email></othercredit>
+<othercredit role="translator"><personname><firstname>悟</firstname> <surname>倉敷</surname><contrib>(日本語訳)</contrib></personname><email>lurdan@gmail.com</email></othercredit>
+<othercredit role="translator"><personname><firstname>修</firstname> <surname>青木</surname><contrib>(日本語訳)</contrib></personname><email>osamu@debian.org</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 <!-- dummy -->
 <!-- dummy -->
