--- maint-guide.ca.dbk-orig	2011-04-01 01:24:17.908310249 +0900
+++ maint-guide.ca.dbk	2011-04-01 01:24:17.920320783 +0900
@@ -19,6 +19,7 @@
 
 
 <!-- BEGIN: Add othercredit for translator via po4a/maint-guide.*.patch -->
+<othercredit role="translator"><personname><firstname>Innocent</firstname> <othername>De</othername> <surname>Marchi</surname><contrib>(Traducción al español)</contrib></personname><email>tangram.peces@gmail.com</email></othercredit>
 <!-- END: Add othercredit for translator via po4a/maint-guide.*.patch -->
 </authorgroup>
 <releaseinfo>Version &docversion;</releaseinfo>
